### PR TITLE
Add GCP Cloud Run example

### PR DESCRIPTION
**Pull request by pawelprazak**
Ported https://github.com/pulumi/examples/tree/master/gcp-ts-cloudrun

### CODE REVIEW

File: examples/gcp-cloudrun/infra/Main.scala [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    docker.ImageArgs(
+      imageName = imageFullName,
+      build = docker.inputs.DockerBuildArgs(
+        context = p"../${appName}",
+        platform = "linux/amd64" // Cloud Run only supports linux/amd64
+      )
+    )
+  )
+
+  // Deploy to Cloud Run. Some extra parameters like concurrency and memory are set for illustration purpose.
```
Comment (lbialy) [inline examples/gcp-cloudrun/infra/Main.scala:42]: Some additional parameters such as concurrency and memory are set for illustration purposes.

Comment (lbialy) [reply examples/gcp-cloudrun/infra/Main.scala:42]: typo

**Review by lbialy — CHANGES_REQUESTED** [review]
minor nitpick only

File: examples/gcp-cloudrun/infra/Main.scala (hunk 1 above)
Comment (pawelprazak) [reply examples/gcp-cloudrun/infra/Main.scala]: done

**Comment by pawelprazak** [comment]
I've fixed the typo.

**Review by lbialy — APPROVED** [review]

**Pull request merged**